firstvoices/backend/serializers/character_serializers.py

```diff
@@ -0,0 +1,55 @@
+from rest_framework import serializers
+
+from backend.models.characters import Character, CharacterVariant, IgnoredCharacter
+from backend.models.dictionary import DictionaryEntry
+from backend.serializers.dictionary_serializers import DictionaryEntryDetailSerializer
+from backend.serializers.fields import SiteHyperlinkedIdentityField
+
+
+class IgnoredCharacterSerializer(serializers.ModelSerializer):
+    url = SiteHyperlinkedIdentityField(view_name="api:ignored_characters-detail")
+    site = serializers.StringRelatedField()
+
+    class Meta:
+        model = IgnoredCharacter
+        fields = ["url", "id", "title", "site"]
+
+
+class CharacterVariantSerializer(serializers.ModelSerializer):
+    base_character = serializers.StringRelatedField()
+
+    class Meta:
+        model = CharacterVariant
+        fields = ["id", "title", "base_character"]
+
+
+class RelatedDictionaryEntrySerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
I kept this code in to build on in the future, but it is unused and can be removed if required

Reply:
I'm on team "when in doubt, leave it out" for things like this.